.github/pull_request_template.md

```diff
@@ -6,4 +6,4 @@
 - [ ] Copy the short description of the proposal as the description of the PR
 - [ ] Remove this checklist section
 - [ ] Submit your PR and the labs stewards will review your proposal
-
+- [ ] Ask your sponsor confirm sponsorship to the stewards (eg; with a comment on the PR)
```

Comment:
I would make this "Ask your sponsor to confirm sponsorship by stating so in a comment to the PR".

Reply:
I concur with @lehors

Reply:
I updated the wording